### PR TITLE
Epic borders and thank you test

### PR DESCRIPTION
**Pull request by jranks123**
## What does this change?

New epic test, testing a) adding a border and an info logo to the epic and b) the impact of thanking our readers in the epic copy.

| Control | control|
|---------|--------|
| Borders - mobile | high |
| Borders - desktop | high |
|   Thankyou message      |  image   |

@guardian/contributions

### CODE REVIEW

**Comment by ionamckendrick** [comment]
We're missing the testimonial in the thank you variant!

**Comment by jranks123** [comment]
@ionamckendrick fixed

**Review by svillafe — APPROVED** [review]
I would rethink the variables `p1`, `p2`, etc but it is just a detail.

File: static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-epic-border-thankyou.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    audienceOffset: 0,
+
+    variants: [
+        {
+            id: 'control',
+            products: [],
+        },
+        {
+            id: 'border',
+            products: [],
```
Comment (Mullefa) [inline static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-epic-border-thankyou.js]: How come we haven't listed any products here?

File: static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-epic-border-thankyou.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                            supportUrl: variant.options.supportURL,
+                        }),
+                        testimonialBlock: variant.options.testimonialBlock,
+                    });
+                },
+            },
+        },
+        {
+            id: 'thankyou',
+            products: [],
```
Comment (Mullefa) [inline static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-epic-border-thankyou.js]: How come we haven't listed any products here?

**Review by ionamckendrick — COMMENTED** [review]
Great!

**Pull request merged**